.github/workflows/build.yml

```diff
@@ -39,28 +39,25 @@ jobs:
       matrix:
         php: [ 8.1, 8.2, 8.3, 8.4 ]
         composer-flags: [ "" ]
-        symfony-version: [ "" ]
         include:
           - php: 8.1
-            symfony-version: '^5.4'
+            composer-flags: "--with=symfony/yaml:^5.4"
           - php: 8.1
-            symfony-version: '^6.4'
+            composer-flags: "--with=symfony/yaml:^6.4"
     steps:
       - uses: actions/checkout@v4
-
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
+      - uses: shivammathur/setup-php@v2
```

Comment:
I would prefer to keep the blank line between each step, I think that's more readable.

Reply:
I mainly did that to be consistent with the other jobs.

However, I find it less readable with the blank lines.. this is essentially a build script, so it's kinda weird to have each line in a script separate by a blank line. I would rather leave that to separate jobs.

Additionally, I also remove the step names since they were mostly repeating what the step did almost verbatim.

Reply:
No issue with removing the step names.

The difference with a build script is you would usually indent arguments to a command that spanned multiple lines. With an actions yaml file they are on the same indentation so the only marker for the start of a new step is the `-` at the start of the line. Not a huge issue if things are just a one-line `run` but as soon as you need to introduce `if`, `env`, etc then it becomes harder to see imo.

Reply:
I think I'd still keep it like that to encourage a 'good' habit in the sense that readers should expect the dash to indicate a new command, rather than a blank line which could mean anything (keep in mind that a long string could be split into multiple lines, including blank lines).
```yaml
- run: |
    this is the

    same command
```
Although my main point would be that conciseness in terms of whitespace tends to be more understandable and show the bigger picture.